index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <nav>
+        <div id="header">
+            <div class="mobile-header">
+                 <div id="primary-logo">
+                     <div id="mobile-logo">
+                     </div>
+                     <div id="logo"></div>
+                 </div>
+                 <div id="mobile-menu">
+                     <!-- <img src="images/ic_menu.png" id="menu-icon" alt="menu-icon" > -->
```

Comment:
- [ ] I think you forgot to remove this commented code as requested by the previous reviewer. Please ensure to do that to make your code look more professional. :ok_hand: